man/systemd.network.xml

```diff
@@ -232,6 +232,20 @@
           the network otherwise.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>Unmanaged=</varname></term>
+        <listitem>
+          <para>A boolean. When <literal>yes</literal>, no attempts are
+          made to bring up or configure matching links, equivalent to
+          when there are no matching network files. Defaults to
+          <literal>no</literal>.</para>
+          <para>This is useful for preventing later matching network
+          files from interfering with certian interfaces that are fully
+          controlled by other applications, such as virtual Ethernet
+          devices that are created and managed internally by sandboxing
+          applications.</para>
```

Comment:
I'm afraid that "such as" explanation does not really explain anything. Can you try to rephrase that, or leave it out?